src/helics/core/networkDefaults.hpp

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright � 2017-2019,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC.  See
+the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
+#pragma once
+
+// For ZMQ Comms
+constexpr int DEFAULT_ZMQ_BROKER_PORT_NUMBER = 23404;
+
+constexpr int BACKUP_ZMQ_BROKER_PORT_NUMBER = 23406;
+
+constexpr int DEFAULT_ZMQSS_BROKER_PORT_NUMBER =
+  23414;  // Todo define a different port number (change in HELICS 3.0)
```

Comment:
What's the problem with this port number?

Reply:
It is right now in the range for the auto allocation of ZMQ brokers.  I would like a little more separation between them, but I don't want to change it until 3.0 as that could cause some issues between the versions.